test/Test/Migration.hs

```diff
@@ -12,125 +12,121 @@ module Test.Migration
   ) where
 
 import Test.Tasty (TestTree, testGroup)
-import Test.Tasty.HUnit (testCase)
 import Util.Named
 
-import Lorentz
+import qualified Lorentz as L
+import Lorentz.Base ((#))
+import Lorentz.Coercions (checkedCoerce)
 import Lorentz.Contracts.TZBTC as TZBTC
 import Lorentz.Contracts.TZBTC.Preprocess
 import qualified Lorentz.Contracts.TZBTC.Types as TZBTCTypes
 import Lorentz.Contracts.TZBTC.V0 (StoreTemplateV0)
 import Lorentz.Contracts.Upgradeable.Common (UContractRouter, mkUContractRouter)
-import Lorentz.Test.Integrational
+import Lorentz.Instr as Lorentz (drop, nil, pair)
+import Lorentz.UParam (mkUParam)
 import Lorentz.UStore.Migration
+import Michelson.Runtime.GState (genesisAddress3)
+import Morley.Nettest
+import Morley.Nettest.Tasty
+import Tezos.Address (Address)
 
 import Test.TZBTC (dummyV1Parameters)
 
 {-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
 
-originateContract
-  :: IntegrationalScenarioM (TAddress (Parameter TZBTCv0))
-originateContract = lOriginate tzbtcContract "TZBTC Contract" (mkEmptyStorageV0 ownerAddress) (toMutez 1000)
 
 -- Test that all administrative endpoints can only be called as owner
 test_ownerCheck :: TestTree
 test_ownerCheck = testGroup "TZBTC contract migration endpoints test"
-  [  testCase "Test call to administrative endpoints are only available to owner" $
-      integrationalTestExpectation $ do
-        -- Originate a V0 contract
-        v0 <- originateContract
-        err <- expectError . withSender bob $
-          lCallDef v0 $ fromFlatParameter $ EpwBeginUpgrade (#current .! 0, #new .! 1)
-        lExpectCustomError_ #senderIsNotOwner err
-  , testCase "Test call to `ApplyMigration` endpoints are only available to owner" $
-      integrationalTestExpectation $ do
-        v0 <- originateContract
-        err <- expectError . withSender bob $
-          lCallDef v0 $ fromFlatParameter $ EpwApplyMigration (checkedCoerce migrationScriptV1)
-        lExpectCustomError_ #senderIsNotOwner err
-
-  , testCase "Test call to `SetCode` endpoints are only available to owner" $
-      integrationalTestExpectation $ do
-        v0 <- originateContract
-        err <- expectError . withSender bob $
-          lCallDef v0 $ fromFlatParameter $ EpwSetCode emptyCode
-        lExpectCustomError_ #senderIsNotOwner err
-
-  , testCase "Test call to `FinishUpgrade` endpoints are only available to owner" $
-      integrationalTestExpectation $ do
-        v0 <- originateContract
-        err <- expectError . withSender bob $
-          lCallDef v0 $ fromFlatParameter $ EpwFinishUpgrade
-        lExpectCustomError_ #senderIsNotOwner err
+  [ nettestScenarioCaps "Test call to administrative endpoints are only available to owner" $ do
+      nettestAddr :: Address <- resolveNettestAddress
```

Comment:
Mm, maybe it would be not too hard to switch to the recent `master` at once?

`resolveNettestAddress` is dead for some time already, probably better use just one more `newAddress` here.

Same for `AddressResolved`, and without it `:: Address` probably would be unnecessary.

Reply:
yep, that's the plan

Reply:
so don't review yet :nabros:

Reply:
> resolveNettestAddress is dead for some time already

Btw, is there a way to get the current `Moneybag` address?